requirements.txt

```diff
@@ -1,3 +1,4 @@
+-e .
```

Comment:
My feeling is this should never happen. Why is this useful?

Reply:
I added that in there because the tests were trying to import kp_registry, which actually means it could go into the requirements-test.txt. If you're just saying to not add this into the requirements.txt but that it still needs to be done, we could add it to the readme.

Reply:
Yeah, I've run into this before. It's only an issue for testing, and only it you're using pytest directly (without tox or, I think, travis), and only if your path is set up in a particular way. So I don't think it should be added to requirement-test.txt either. The key is just to make sure that the project directory is on your path. One option is to run `PYTHONPATH=. pytest...`. Of course you can also `pip install -e .` in your dev environment, but it is not a general requirement for everyone.